egui_demo_lib/src/apps/demo/plot_demo.rs

```diff
@@ -421,8 +421,11 @@ impl Widget for &mut ItemsDemo {
         };
 
         let texture: &egui::TextureHandle = self.texture.get_or_insert_with(|| {
-            ui.ctx()
-                .load_texture("plot_demo", egui::ColorImage::example())
+            ui.ctx().load_texture(
+                "plot_demo",
+                egui::ColorImage::example(),
+                egui::TextureFilter::default(),
```

Comment:
```suggestion
                Default::default(),
```

this is shorter and more future proof (i.e. will work even if we change the name/type of this argument)